src/main/webapp/app/exam/manage/exams/exam-update.component.ts

```diff
@@ -238,7 +292,7 @@ export class ExamUpdateComponent implements OnInit {
      */
     get validateWorkingTime(): boolean {
         if (this.exam.testExam) {
-            if (this.exam.workingTime === undefined || this.exam.workingTime < 1) {
+            if (!this.exam.workingTime || this.exam.workingTime < 1) {
```

Comment:
I would use the old version, since `!this.exam.workingTime` also is true for `!this.exam.workingTime = 0`. In the end the condition works the same, but it gets more complicated when thinking about the two parts of the condition and when they are true. The first part is really just meant to catch undefined

Reply:
Yea you're totally right